### DISCUSSION TITLE
Will Copilot X be free for students and educators like vanilla Copilot?

### DISCUSSION THREAD

**Discussion by davidleejy** [community/community#51696] [Copilot Conversations]
### Select Topic Area

Question

### Body

Copilot has been quite useful in my experience. Will Copilot X be free for students and educators like vanilla Copilot?

**Comment by Akash1134** [comment]
Hi @davidleejy ,  GitHub Copilot X is currently a representation of GitHub’s vision for the future rather than an available product offering of GitHub Copilot. As we continue to design, test, and build features that fall into the GitHub Copilot X vision, we are also taking the time to determine the best way to provide them to our customers. 

While we are unable to guarantee access to each feature, you can submit your interest by [joining the waitlist](https://github.com/features/preview) to preview the features that you are interested in. Joining one waitlist does not confirm your interest in more than one feature, so you must join each separately. Please note we currently don’t have guaranteed timelines for access.

Once you get access to preview a feature, please keep in mind that what you are getting access to is considered beta or a technical preview, meaning the feature may still have some kinks to work out.

For more details, please check out our [FAQs](https://github.com/features/preview/copilot-x#:~:text=Frequently%20asked%C2%A0questions)
